### PR TITLE
Get started with IP address management

### PR DESCRIPTION
**Pull request by bmckercher123**
New How to.

- Can merge at any time.
- Approved by Casey

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 424f10ccee03f953d01c80b5d02c2ae26f324e66

https://deploy-preview-2845--calico.netlify.com

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 424f10ccee03f953d01c80b5d02c2ae26f324e66

https://deploy-preview-2845--calico.netlify.com

File: master/networking/get-started-ip-addresses.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+### Above and beyond
+
+- [IP Pool]({{site.baseurl}}/{{page.version}}/reference/resources/ippool)
+
+There are several other ways to leverage Calico IPAM including:
+
+- [Assign addresses based on topology]({{site.baseurl}}/{{page.version}}/networking/assigning-ip-addresses-topology)
+- [Use a specific address for a pod]({{site.baseurl}}/{{page.version}}/networking/use-specific-ip)
+- [Migrate from one IP to another]({{site.baseurl}}/{{page.version}}/networking/changing-ip-pools)
```
Comment (caseydavenport) [inline master/networking/get-started-ip-addresses.md]: from one IP pool to another

File: master/networking/get-started-ip-addresses.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+#### IPAM in Kubernetes 
+
+Kubernetes uses IPAM plugins to allocate and manage IP addresses assigned to pods. Different IPAM plugins provide different feature sets. Calico provides its own IPAM plugin called **calico-ipam** which is designed to work well with Calico and includes a number of features. 
+
+#### Calico IPAM
+
+The **calico-ipam** plugin uses Calico’s IP pool resource to control how IP addresses are allocated to pods within the cluster. This is the default plugin used by most Calico installations.
+
+By default, Calico uses a single IP pool for the entire Kubenretes pod CIDR, but you can divide the pod CIDR into several pools. You can assign separate IP pools to particular selections of **nodes**, or to teams, users, or applications within a cluster using **namespaces**.
```
Comment (lmm) [inline master/networking/get-started-ip-addresses.md]:
```suggestion
By default, Calico uses a single IP pool for the entire Kubernetes pod CIDR, but you can divide the pod CIDR into several pools. You can assign separate IP pools to particular selections of **nodes**, or to teams, users, or applications within a cluster using **namespaces**. 
```

**Pull request merged**